packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -102,7 +102,7 @@ class BuildWebCommand extends BuildSubCommand {
     argParser.addSeparator('WebAssembly compilation options');
     argParser.addFlag(
       FlutterOptions.kWebWasmFlag,
-      help: 'Compile to WebAssembly rather than JavaScript.\n$kWasmMoreInfo',
+      help: kWasmCompile,
```

Comment:
rather than making the help text dry, could we make this whole flag dry by adding a helper method for it to flutter_command.dart, such as: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L519

Reply:
good idea! done